docs/source/user_guide/accessing_results.rst

```diff
@@ -4,39 +4,42 @@
 Accessing Results
 *****************
 
-The :py:class:`Solution <ansys.dpf.post.dpf_solution.DpfSolution>`
+The :class:`Solution <ansys.dpf.post.dpf_solution.DpfSolution>`
 object is the entry point for browsing the contents of a result file
```

Comment:
results file or results file? Earlier, it had an "s" at the end.

Reply:
Good question. You pick and let's just be consistent.